packaging/linux_simple/ungoogled-chromium.yml

```diff
@@ -0,0 +1,30 @@
+# # TODO: Add https://www.i-dont-care-about-cookies.eu/ - https://github.com/uBlockOrigin/uBlock-issues/issues/196#issuecomment-417963577
```

Comment:
I believe this TODO should be removed since it goes against the objective of sticking to the default Chromium experience.

Reply:
Yes indeed i just forget to remove it was on the original script